modules/javafx.controls/src/shims/java/javafx/scene/control/skin/TableHeaderRowShim.java

```diff
@@ -32,4 +34,11 @@ public static TableColumnHeader getColumnHeaderFor(TableHeaderRow tr, final Tabl
         return tr.getColumnHeaderFor(col);
     }
 
+    public static ContextMenu getColumnPopupMenu(TableHeaderRow tableHeaderRow) {
+        return tableHeaderRow.getColumnPopupMenu();
```

Comment:
Minor: I would keep `tr` instead of `tableHeaderRow` for consistency with already existing static method.

Reply:
I would too - avoids wrapping the line, and there is no need to spell the name for a one line method.

Reply:
I am more a fan of descriptive variable names. I know this is a one-line method - but on the other hand `tr` is just not a good understandable name, even for a very short method.
Since you read code more often than you write it, I'm for keeping the descriptive name, also in this simple case :)